libs/lwt_wrapper/dune

```diff
@@ -0,0 +1,8 @@
+(library
+ (public_name lwt_wrapper)
+ (wrapped false)
+  (libraries
+    lwt
+  )
+ (virtual_modules lwt_wrapper)
```

Comment:
It may be good to have a readme in this directory or a comment in the dune file here explaining what is going on with the virtual module, why it is being used, and how it functions.

Reply:
maybe because this is the first use of virtual_modules in Semgrep, maybe a small comment before giving a high-level overview of the feature would be useful, and which concrete problem in Semgrep (and LSP.js) it solves.

Reply:
Yes will do